README.md

```diff
@@ -1,24 +1,109 @@
-**Phorge** is a collection of web applications which help software companies build better software.
+# Phorge
+*ThoughtMachine.* Most of the changes we've
+made for ourselves over the years
+except for a few we no longer need
+or just don't fit in style wise, but
+including some that are hard to find
+because either people forgot about
+them or simply because they haven't
+been released yet, a few we really love,
+one we think is just ok, some we did
+for free, some we did for money, some
+for ourselves without permission and
+some for friends as swaps but never on
+time and always at our office in ~~Old Street~~ Herbrand Street.
 
-Phorge is a community-maintained fork of [Phabricator](http://phabricator.org).
+# About this repository
 
-Phorge includes applications for:
+Thought Machine's fork of [Phorge](https://github.com/phorgeit/phorge). Phorge itself is a community-maintained
+fork of [Phabricator](http://phabricator.org), developed and maintained by [The Phorge Team](https://phorge.it).
 
-  - reviewing and auditing source code;
-  - hosting and browsing repositories;
-  - tracking bugs;
-  - managing projects;
-  - conversing with team members;
-  - assembling a party to venture forth;
-  - writing stuff down and reading it later;
-  - hiding stuff from coworkers; and
-  - also some other things.
+Our changes are maintained on the `tm-master` branch.
 
+We probably won't accept any pull requests for new features as this repository
+and the associated changes are purely quality-of-life changes for our own
+installation. Changes that improve the style or robustness will probably be
+accepted though!
```

Comment:
I don't think we should encourage this unless the style or robustness improvements are for our own code - we want to keep this as close to upstream as possible, so those changes should go upstream.